var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -105,6 +105,8 @@ class PyNumpy(PythonPackage):
     patch('add_fj_compiler3.patch', when='@1.14.0:1.18.5%fj')
     patch('add_fj_compiler4.patch', when='@:1.13.3%fj')
 
+    patch('check_executables.patch')
```

Comment:
Does this patch apply cleanly to all versions? Has this patch been submitted upstream?

Reply:
@adamjstewart 
Fixed to apply patch to all versions.
We haven't submitted it to upstream yet, so our team will consider it.

Reply:
Patches look good now. I would still like to see this submitted to the numpy folks so they can comment on whether or not this is the correct approach.

Reply:
@adamjstewart I submitted this issue.
https://github.com/numpy/numpy/issues/18411